deploy/static/provider/oci/deploy.yaml

```diff
@@ -328,26 +328,27 @@ metadata:
     app.kubernetes.io/instance: ingress-nginx
     app.kubernetes.io/name: ingress-nginx
     app.kubernetes.io/part-of: ingress-nginx
-    app.kubernetes.io/version: 1.7.1
+    app.kubernetes.io/version: 1.8.0
   name: ingress-nginx-controller
   namespace: ingress-nginx
 ---
 apiVersion: v1
 kind: Service
 metadata:
+  annotations:
+    service.beta.kubernetes.io/oci-load-balancer-shape: flexible
+    service.beta.kubernetes.io/oci-load-balancer-shape-flex-max: "100"
+    service.beta.kubernetes.io/oci-load-balancer-shape-flex-min: "10"
   labels:
     app.kubernetes.io/component: controller
     app.kubernetes.io/instance: ingress-nginx
     app.kubernetes.io/name: ingress-nginx
     app.kubernetes.io/part-of: ingress-nginx
-    app.kubernetes.io/version: 1.7.1
+    app.kubernetes.io/version: 1.8.0
   name: ingress-nginx-controller
   namespace: ingress-nginx
-  annotations:
-    service.beta.kubernetes.io/oci-load-balancer-shape: "flexible"
-    service.beta.kubernetes.io/oci-load-balancer-shape-flex-min: "10"
-    service.beta.kubernetes.io/oci-load-balancer-shape-flex-max: "100"
 spec:
+  externalTrafficPolicy: Local
```

Comment:
Other deploy files have `externalTrafficPolicy: Local`, but the original OCI file didn't have this field.

Reply:
FYI, [Oracle Best Practice: Configure externalTrafficPolicy appropriately when exposing applications](https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengbestpractices_topic-Networking-best-practices.htm#:~:text=externalTrafficPolicy%3A%20Local%20(local%20mode)&text=Local%20mode%20removes%20the%20extra,corresponding%20pods%20for%20that%20service.)